doc/glossary.rst

```diff
@@ -76,13 +76,31 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward model
+        The forward model is the numerical representation of a subject's brain,
```

Comment:
The forward solution (abbr. ``fwd``) is the obtained by solving Maxwell equations that describe how the
electromagnetic fields propagate from the brain to the MEG/EEG sensors. It is a linear
operator that requires a conductivity model which can be a sphere or a BEM model.
It captures the relationship between each dipole in the source space and the expected
pattern of activity on the sensors.

We need then entries for "conductivity model" and "BEM"

Reply:
How about this:
```
The forward solution or forward model (abbr. ``fwd``) is a linear
operator capturing the relationship between each dipole location in the
:term:`source space` and the expected pattern of activity on the
sensors corresponding to that dipole (AKA, the "lead fields"). Forward
solutions are obtained by solving the Maxwell equations describing
electromagnetic field propagation; this requires a conductivity model
of the different tissues that the electromagnetic waves must propogate
through in order to reach the sensors (see :term:`boundary element
model <BEM>` and :class:`mne.bem.ConductorModel`).
```